lib/osf-components/addon/components/file-browser/add-new/upload/styles.scss

```diff
@@ -19,3 +19,7 @@
 .CheckIcon {
     color: $brand-success;
 }
+
+.UploadModal {
```

Comment:
There is also the `@fixedWidth` option for OsfDialog, but if it looks too wide using that, I think this width class works too.